rmgpy/data/kinetics/family.py

```diff
@@ -3704,47 +3704,43 @@ def cross_validate(self, folds=5, template_rxn_map=None, test_rxn_inds=None, T=1
                     if entry.parent:
                         entry = entry.parent
 
+                boo = True
```

Comment:
Why `boo`? A more informative name would help. ANd I don't see where `boo` is ever set to anything but True? So what's this infinite loop about? 
Does it mean `while entry.parent:` ?